micro-infra-spring-base/build.gradle

```diff
@@ -31,9 +31,9 @@ dependencies {
     testCompile "org.spockframework:spock-core"
     testRuntime "org.spockframework:spock-spring"
     testCompile "org.springframework:spring-test"
-    testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'org.hamcrest:hamcrest-library:1.4-atlassian-1'
```

Comment:
I think that "References: 1681" reflects to hamcrest-core in general, not that version. I would stick with 1.3 or check if version [2.0.0.0](https://github.com/hamcrest/JavaHamcrest/blob/master/CHANGES.txt) is (not) compatible with other libraries :).